### PR TITLE
test: add spring-web integration test with @PathVariable without configuration

### PR DESCRIPTION
**Pull request by aureamunoz**
Related to issue #3992
~~Requires RESTEasy: 4.4.0.Final (still unreleased)~~

This adds a integration test for spring-web using a method with @PathVariable annotated parameter without value for `name`. It's waiting for the RESTEasy fix in [this PR](https://github.com/resteasy/Resteasy/pull/2177). Could you take a look @geoand or forward to someone else?

### CODE REVIEW

**Comment by geoand** [comment]
This is good!

Once RESTEasy is released with this PR, I will approve it :)

**Comment by emmanuelbernard** [comment]
@asoldano hey, can you provide some visibility on when RESTEasy 4.4.0.Final will be released?

**Comment by asoldano** [comment]
@emmanuelbernard , considering the amount of features we have on master atm, I think it would be doable to release by the end of October (I just need to allocate a bit of time for the JavaEE -> JakartaEE api switch). Would that be ok for you? Feel free to drop me an email for additional details / discussion.

**Comment by geoand** [comment]
@aureamunoz can you please rebase onto master (which has been updated to use `4.4.0.Final` of RESTEasy) and force push ASAP?
We should really have this one for the next release.

Thanmks

**Comment by aureamunoz** [comment]
yes, in progress

**Comment by aureamunoz** [comment]
> @aureamunoz can you please rebase onto master (which has been updated to use `4.4.0.Final` of RESTEasy) and force push ASAP?
> We should really have this one for the next release.
> 
> Thanks

Done

**Comment by geoand** [comment]
Thanks!

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -2421,7 +2421,7 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-            
+
```
Comment (geoand) [inline bom/runtime/pom.xml]: Can we get rid of this change please?

**Comment by geoand** [comment]
@aureamunoz I requested a minor change. Once that is done, we can approve.

File: bom/runtime/pom.xml (hunk 1 above)
Comment (geoand) [reply bom/runtime/pom.xml]: I guess this was added by the formater - let's forget about it

**Review by geoand — APPROVED** [review]
Thanks!

**Pull request merged**